src/marqo/index.py

```diff
@@ -389,19 +453,19 @@ def batch_requests(gathered, doc_tuple):
         batched = functools.reduce(lambda x, y: batch_requests(x, y), deeper, [])
 
         def verbosely_add_docs(i, docs):
-            t0 = datetime.now()
+            t0 = timer()
             if update_method == 'replace':
                 res = self.http.post(path=path_with_query_str, body=docs)
             elif update_method == 'update':
                 res = self.http.put(path=path_with_query_str, body=docs)
             else:
                 raise ValueError(f'Received unknown update_method `{update_method}`. '
                                  f'Allowed update_methods: ["replace", "update"] ')
-            total_batch_time = datetime.now() - t0
+            total_batch_time = timer() - t0
             num_docs = len(docs)
             mq_logger.info(
-                f"batch {i}: ingested {num_docs} docs. Time taken: {total_batch_time}. "
-                f"Average timer per doc {total_batch_time/num_docs}")
+                f"  add_documents batch {i}: added {num_docs} docs. Time taken: {(total_batch_time):.3f}s. "
+                f"Average time per doc {(total_batch_time/num_docs):.3f}s.")
```

Comment:
Add time taken to execute add_docs on Marqo.
Ideally keep the total logging lines to 2 per batch (perhaps one for roundtrip, one for Marqo execution